tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -528,4 +528,11 @@ int main() {
         auto r2 = s | views::drop(evil_convertible_to_difference{});
         assert(ranges::equal(r2, only_four_ints));
     }
+
+    { // GH-3025 <iterator>: ranges::prev maybe ill-formed in debug mode
+        auto r    = views::iota(0ull, 5ull);

```

Comment:
It would be better to add this test to `P0896R4_views_iota` since this involves its usage of integer-class types and has nothing to do with `views::drop`.